Sprint-2/stretch/till.js

```diff
@@ -8,24 +8,44 @@ function totalTill(till) {
   let total = 0;
 
   for (const [coin, quantity] of Object.entries(till)) {
-    total += coin * quantity;
+
+    let coinValue;
+
+    if(coin.endsWith("p")){
+      coinValue = parseInt(coin.replace('p', ""), 10);
```

Comment:
`parseInt(coin)` would also work. May I suggest checking out how `parseInt()` convert a string to an integer?